File: Algorithms/Sorting/Bubblesort/bubblesort.py
def bubble_sort(list):
    unsorted_until_index = len(list)-1  #Track Rightmost index that is not sorted yet
    sorted = False
    while not sorted:
        sorted = True
        for i in range(unsorted_until_index):
            if list[i]> list[i+1]:
                list[i],list[i+1] = list[i+1],list[i]
                sorted = False
        unsorted_until_index -=1
    return list


print(bubble_sort([65,45,11,23,999,34,1,3]))

File: Algorithms/Sorting/Bubblesort/bubblesort.js
function bubble_sort(array){
   let unsorted_last_index = array.length -1
    let sorted = false
    for (let i=0;i<unsorted_last_index;++i){
        sorted = true
        for(let j=0;j<unsorted_last_index-i;++j){
            if(array[j]>array[j+1]){
                [array[j],array[j+1]] =[array[j+1],array[j]]
                sorted = false
            }
        }
        if (sorted)break;
    }
}


let array = [13,88,91,22,1,23,12,3]
bubble_sort(array)
console.log("Sorted Array:",array)